### DISCUSSION TITLE
How to use add_polyhedron() function in class StructureVis()

### DISCUSSION THREAD

**Discussion by thienbinh92** [materialsproject/pymatgen#2554] [Q&A] [answered]
Hi, I am not clear about the meaning of the neighbor parameter in the add_polyhedron() function. It explains "neighbors – Neighbors of the polyhedron (the vertices)." For example, I have an LCO structure. I would like to show polyhedrons for CoO6, not LiO6. How can I define the "neighbors" parameter?
![Screenshot 0004-06-22 at 17 57 16](https://user-images.githubusercontent.com/56110825/174988599-ec2556a8-c35b-43ca-bb3c-13545fe8c489.png)

**Comment by janosh** [comment, marked as answer]
The error message arises because the `neighbors` argument expects an iterable. Try setting `neighbors=[6]`.
